src/tests/Interop/UnmanagedCallersOnly/UnmanagedCallersOnlyTest.cs

```diff
@@ -35,6 +37,20 @@ public static class UnmanagedCallersOnlyDll
         public static extern int PInvokeMarkedWithUnmanagedCallersOnly(int n);
     }
 
+    private const string InvalidCSharpAssemblyName = "InvalidCSharp.dll";
+
+    public static Type GetCallbacksType()
+    {
+        var asm = AssemblyLoadContext.Default.LoadFromAssemblyPath(Path.Combine(Environment.CurrentDirectory, InvalidCSharpAssemblyName));
```

Comment:
A simple `Assembly.Load("InvalidCSharp")` does not work?

Reply:
It does. I have been falling back to ALC usage just because it seems like the clearest way now.

Fixed.

Reply:
`Assembly.Load` is the cleanest way when the assembly is part of your application that should be majority of cases.